integrationTests/chainSimulator/bridge/common.go

```diff
@@ -80,23 +92,16 @@ func deployBridgeSetup(
 	require.Nil(t, err)
 	nonce := uint64(0)
 
-	esdtSafeArgs := "@" + // is_sovereign_chain
-		"@01" + lengthOn4Bytes(len(argsEsdtSafe.IssuePaymentToken)) + hex.EncodeToString([]byte(argsEsdtSafe.IssuePaymentToken)) + // token identifier
-		"@01" + lengthOn4Bytes(len(argsEsdtSafe.ChainPrefix)) + hex.EncodeToString([]byte(argsEsdtSafe.ChainPrefix)) // prefix
-	esdtSafeAddress := chainSim.DeployContract(t, cs, ownerAddrBytes, &nonce, systemContractDeploy, esdtSafeArgs, esdtSafeWasmPath)
+	esdtSafeAddress := deployEsdtSafeContract(t, cs, ownerAddrBytes, &nonce, systemContractDeploy, argsEsdtSafe)
 
 	feeMarketArgs := "@" + hex.EncodeToString(esdtSafeAddress) + // esdt_safe_address
-		"@000000000000000005004c13819a7f26de997e7c6720a6efe2d4b85c0609c9ad" + // price_aggregator_address
-		"@" + hex.EncodeToString([]byte("USDC-350c4e")) + // usdc_token_id
-		"@" + hex.EncodeToString([]byte("WEGLD-a28c59")) // wegld_token_id
```

Comment:
Is it ok we don't have any fee market address set and tested anymore?

Reply:
Those 3 args were are for a feature not implemented yet and are not used at the moment. We decided to remove them from `init` contract function. We'll see later if you add them back or there will be a different implementation.